djautotask/api.py

```diff
@@ -22,6 +22,10 @@
 NO_RECORD_ERROR_MESSAGE = \
     'No message. No matching records found. The logged in Resource may not ' \
     'have the required permissions to delete the data.'
+CACHE_KEYS = {
+    'url': 'zone_info_url',
+    'webUrl': 'zone_info_webUrl'
```

Comment:
What's the benefit of putting these into a dict to access them as different names? On line 425 you go back to just using 'zone_info_url' again anyway. If there isn't a benefit to this you can just remove it and use 'zone_info_url' and 'zone_info_webUrl'.

Reply:
Okay, just a min. The AT response is as follows:

![image](https://user-images.githubusercontent.com/73486314/140561726-9892d625-6c95-4cca-abce-b62931e018bf.png)

So, it could be better to make consistency using `url` and `webUrl`

Reply:
Fix applied, and review is ready again